### ISSUE TITLE
Fatal errors are different on HHVM & PHP7

### ISSUE THREAD

**Issue by Jean85** [facile-it/paraunit#12]
https://travis-ci.org/facile-it/paraunit/jobs/81814347

PHP7 and HHVM handle fatal errors as errors.... We should fix the parsing (and the build!)

**Comment by Jean85** [comment]
For fixing the build, we need a non-recoverable fatal error for PHP7 and HHVM... Maybe a memory exausted error?

**Comment by fntlnz** [comment]
Just for the records

```
docker run --rm fntlnz/php:5.6.13 php  -r '$arr=[]; while(true) {$arr[]="yo";}'

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32 bytes) in Command line code on line 1
```

```
docker run --rm fntlnz/php:7.0.0RC3 php  -r '$arr=[]; while(true) {$arr[]="yo";}'

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 134217736 bytes) in Command line code on line 1
```

**Comment by Jean85** [comment]
I imagined that something like that would work! :facepunch:

**Comment by Jean85** [comment]
Build fixed with 84d43a675e4d8dc7ab8cd616904dfed8fce84697!

**Comment by Jean85** [comment]
I don't think we need more handling here, PHPUnit should shield us from the PHP5-7 differences about fatal errors and so on...
